Gulpfile.js

```diff
@@ -70,3 +77,96 @@ gulp.task("watch", ["build"], function() {
     gulp.start("build");
   });
 });
+
+gulp.task("build-babel-standalone", cb => {
+  pump(
+    [
+      gulp.src(__dirname + "/packages/babel-standalone/src/index.js"),
+      webpackBuild(),
+      gulp.dest(__dirname + "/packages/babel-standalone"),
+      uglify(),
+      rename({ extname: ".min.js" }),
+      gulp.dest(__dirname + "/packages/babel-standalone"),
+    ],
+    cb
+  );
+});
+
+function webpackBuild() {
+  let version = require("./packages/babel-core/package.json").version;
+
+  // If this build is part of a pull request, include the pull request number in
+  // the version number.
+  if (process.env.CIRCLE_PR_NUMBER) {
+    version += "+pr." + process.env.CIRCLE_PR_NUMBER;
+  }
+
+  const config = {
+    module: {
+      rules: [
+        {
+          test: /\.js$/,
```

Comment:
is this the same thing as below? it seems that way except the stage-0 part? And should we use loose mode?